kogito-business-rules-springboot/README.md

```diff
@@ -58,8 +88,26 @@ To make use of this application it is as simple as putting a sending request to
 
 ```
 
-Complete curl command can be found below:
+Complete curl command can be found below for adults:
 
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"person" : {"name" : "john", "age" : 20}}' http://localhost:8080/persons
 ```
+
+After the Curl command you should see a similar console log
+```
+{"id":"fd4f629d-6822-4ca2-a8a6-a74f5f81e83d","person":{"name":"john","age":20,"adult":true}} 
+```
+
```

Comment:
Explain here that there is no task running because it is an adult:

`curl http://localhost:8080/persons/{uuid}/tasks`